plotly/offline/offline.py

```diff
@@ -193,7 +208,7 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
 
     plot_html, plotdivid, width, height = _plot_html(
         figure_or_data, show_link, link_text, validate,
-        '100%', 525, global_requirejs=True)
+        '100%', 525, global_requirejs=__PLOTLY_USE_CDN)
```

Comment:
@yankev : Should not this be opposite? I havent looked at the complete code, but if `__PLOTLY_USE_CDN` is `True`, should not `global_requirejs=False`?